apps/explorer/src/app/components/txs/details/tx-proposal.tsx

```diff
@@ -57,6 +57,7 @@ export function proposalTypeLabel(terms?: ProposalTerms): string {
   } else if (has(terms, 'updateAsset')) {
     return t('Update asset proposal');
   } else if (has(terms, 'newMarket')) {
+    // @ts-ignore TODO: figure out this type error, changes will be present
```

Comment:
chore (non-blocking): this is optionally chained so based on this message the ts-ignore can be removed

```suggestion
    // TODO: figure out this type error, changes will be present
```

Reply:
Yes the optional chain should protect when the code runs but running build-spec will fail